server/routerlicious/packages/tinylicious/src/routes/summaries/getLatestSummaryApi.ts

```diff
@@ -0,0 +1,160 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Router } from "express";
+import type { ICommitDetails } from "@fluidframework/gitresources";
+import type { ITenant, ITenantManager } from "@fluidframework/server-services-core";
+import { getParam } from "../../utils";
+
+export function createGetLatestSummaryApiRoute(
+	tenantManager: ITenantManager,
+	router: Router,
+): Router {
+	router.get("/latest/:tenantId/:documentId", (request, response) => {
+		console.log("received summary get request");
+		const documentId = getParam(request.params, "documentId");
+		const tenantId = getParam(request.params, "tenantId");
+
+		const tenantManagerP = tenantManager.getTenant(tenantId, documentId);
+
+		tenantManagerP
+			.then(async (tenant) => {
+				const apiResponse: GetSummaryApiResponse = await getLatestSummary(
+					tenant,
+					documentId,
+				);
+				return response.status(200).json(apiResponse);
+			})
+			.catch((error) => {
+				console.log(error);
+				response.status(400).json(error);
+			});
+	});
+
+	return router;
+}
+
+/**
+ * Object returned by the getSummary route
+ */
+export interface GetSummaryApiResponse {
+	/**
+	 * Information about the commit in which the DDS data is from.
+	 * This is useful for seeing if the data is from a newly initialized (app) or a summary of a app and if so how new it is.
+	 *
+	 * @example
+	 * A newly created document will have the following in the field gitCommit.commit.message:
+	 * ```
+	 * "New document\n"
+	 * ```
+	 *
+	 * 	 * @example
+	 * A summary created from a document will have a string *similar* to the following in gitCommit.commit.message:
+	 * ```
+	 * "Summary @22:20\n"
+	 * ```
+	 *
+	 *
+	 */
+	gitCommit: ICommitDetails;
```

Comment:
For a final version, I'd hesitate to expose this in this form. The fact we use git for the summaries should be a non-exposed implementation detail, I think.